configure.ac

```diff
@@ -560,6 +560,7 @@ dnl Checks for library functions.
 AC_CHECK_FUNCS(putenv)
 AC_CHECK_FUNCS(mempcpy)
 AC_CHECK_FUNCS(fdatasync)
+AC_CHECK_DECLS(fdatasync, [], [], [#include <unistd.h>])
```

Comment:
`fdatasync()` was just replaced with `fsync()` in 6151ac9a298a9fe560cf8f899dc0b0e67453446c.